### ISSUE TITLE
Upload failiure

### ISSUE THREAD

**Issue by josedesousa** [StudioMercury/digital-publishing-tools-for-wordpress#2]
Metadata & Content (thumbnails) are uploading perfectly. However the articles fail to go over.

This issue has been replicated in both Apache and nGinx environments.

![fail](https://cloud.githubusercontent.com/assets/12964117/9352560/157603bc-465a-11e5-80ac-5f5720f67337.jpg)

**Comment by josedesousa** [comment]
The following pull request rectifies this problem.

https://github.com/StudioMercury/digital-publishing-tools-for-wordpress/pull/3

However, once a folio has been successfully uploaded, attempting to update either metadata or  content results in the following error:

![screen shot 2015-08-24 at 14 11 30](https://cloud.githubusercontent.com/assets/12964117/9440887/99540d8c-4a6a-11e5-9693-6245e807ce41.png)

**Comment by StudioMercury** [comment]
Hi @josedesousa-tcc, 

We just pushed a new update today that addresses some bugs. We're recommending people
- delete the old plugin and re-upload the new one (v2.0.6)
- if you moved the "publish-templates" folder to your theme folder please replace it with the new "publish-templates" folder inside of the new v.2.0.6 release. 
- in Wordpress click Settings > Permalinks and then click the Save changes button. 

Thank you,
Studio Mercury

**Comment by toby1kenobi** [comment]
I have the new plugin (2.0.6) and have created an article in the Articles tab of the plugin section (/wp-admin/admin.php?page=dps_folio_author#/article/list). 

Under Properties I added Article Name, Article Title, Article Image. I then went to the article's page in WordPress itself, and added body copy. 

I then went back to the plugin and did Cloud / 'Create in the Cloud'. I then tried Cloud / Publish. I get the attached error:

![dps](https://cloud.githubusercontent.com/assets/3606161/10226119/a1e46b20-685d-11e5-8d83-d9f770dbe008.png)

Should I be able to get this to work?
